tests/immu/test_session.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2022 CodeNotary, Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#       http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from immudb import constants
+from tests.immuTestClient import ImmuTestClient
+import time
+
+
+class TestSessionTransaction:
+
+    def test_simple_unmanaged_session(self, wrappedClient: ImmuTestClient):
+        if(not wrappedClient.serverHigherOrEqualsToVersion("1.2.0")):
+            pytest.skip("Version too low")
+        txInterface = wrappedClient.client.openSession("immudb", "immudb", b"defaultdb")
```

Comment:
Why username and password are strings and database name is byte array?

In golang SDK it's actually the opposite - username and password are byte arrays and the database name is a string (https://pkg.go.dev/github.com/codenotary/immudb@v1.3.0/pkg/client#ImmuClient)

Reply:
Decided to allow str and bytes, so we are backward compatible
done in https://github.com/codenotary/immudb-py/pull/46/commits/037d9354f465c5452c5e0a1065be529c07449f40